app/models/customer.rb

```diff
@@ -1,6 +1,17 @@
 class Customer < ApplicationRecord
   has_many :invoices
+  has_many :merchants, through: :invoices
```

Comment:
thanks for adding these in on this branch, glad we caught that, could have been one of those situations where we run into more and more problems because of these, we should be good now